presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleNodePartitioningProvider.java

```diff
@@ -44,18 +42,11 @@ public BlackHoleNodePartitioningProvider(NodeManager nodeManager)
     public ConnectorBucketNodeMap getBucketNodeMap(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorPartitioningHandle partitioningHandle)
     {
         // create one bucket per node
-        return createBucketNodeMap(nodeManager.getRequiredWorkerNodes().size());
-    }
-
-    @Override
-    public ToIntFunction<ConnectorSplit> getSplitBucketFunction(
-            ConnectorTransactionHandle transactionHandle,
-            ConnectorSession session,
-            ConnectorPartitioningHandle partitioningHandle)
-    {
-        return value -> {
-            throw new PrestoException(NOT_SUPPORTED, "Black hole connector does not supported distributed reads");
-        };
+        return createBucketNodeMap(
```

Comment:
consider making this change in the commit that introduced `ConnectorBucketNodeMap`

Reply:
Will squash it before merge